### DISCUSSION TITLE
Read data from directus_system collections in flows

### DISCUSSION THREAD

**Discussion by Borty97** [directus/directus#24052] [Feature Requests]
### Summary

Add the option to read data from `system_collections` on the flows

### Basic Example

Trigger actions according to system collections in Directus, like revisions, activity, etc. For example, do something with data and delta values from a revision (like create a new item in a collection, send emails with changes, etc)

### Motivation

Automate flows and use the system collections to be the trigger to do something great

### Detailed Design

Just read data from other collections. Should be great

### Must Have:
- Read data from directus_collection

### Drawbacks

I think it's easy to implement, and no great costs will occur

### Alternatives

Make an API request with a webhook action in flows, but can not use the `token` to make the call, because session is not started there. We should login, save the token and then make the request

### Adoption Strategy

It will not affect. It's just an improve and will not affect to collections, database or anything

**Comment by rijkvanzanten** [comment]
I noticed you touched on activity and revisions specifically, do you have any use cases you were trying to achieve that were impossible before? The operations for system databases have to be implemented with a bit more care as there's side effects on those collections that affect system behavior. Trying to learn a bit more about your use cases so we can prioritize and subdivide the individual tasks involved here 🙂

**Comment by IamSAB** [comment]
A use case from my side would be to assign a new role or a policy based on a custom collection items.

In my case, I would like to have a responsibility collection where I define which user has which responsibility in a time frame. A flow would get all responsibilities and check which one are active and assign or remove policies to users based on assign responsibilities.
In a gantt view I would have a good overview of what responsibility a user had/has/going to have at which time. With the flow running I do not have to care about assigning policies. That way I could plan accesibility.
